courses/free-courses-en.md

```diff
@@ -1168,6 +1168,7 @@
 * [Complete Machine Learning in Python playlist](https://www.youtube.com/playlist?list=PLZoTAELRMXVPBTrWtJkn3wWQxZkmTXGwe) - Krish Naik
 * [Deep Learning Fundamentals](https://cognitiveclass.ai/courses/introduction-deep-learning) - DeepLearning.TV (cognitiveclass.ai)
 * [Google's Machine Learning Crash Course](https://developers.google.com/machine-learning/crash-course/)
+* [Hacker's guide to learning model](https://youtu.be/jkrNMKz9pWU?si=Jv-dDoLIQ8gYLANB)
```

Comment:
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes 
No shortened links.

Reply:
```suggestion
* [Hacker's guide to learning model](https://www.youtube.com/watch?v=jkrNMKz9pWU)
```